src/lib.rs

```diff
@@ -295,18 +305,19 @@ if_everything! {
         Unknown(u64),
     }
 
-    // TODO: this could avoid std using peek_bytes
```

Comment:
Uh this is awesome, this todo has been here a very long time, and I really appreciate you fixing it! Great work :)